src/p2p/ModeSystemFuncs.ts

```diff
@@ -698,5 +723,5 @@ export function getExpiredRemovedV3(
     insertSorted(removed, node.id)
   }
 
-  return { problematic: problematicNodes.length, expired: numExpiredNodes, removed }
+  return { problematic: numProblematicRemovals, expired: numExpiredRemovals, removed }
```

Comment:
note: we are reporting the number of node we removed as a result of being expired or problematic. 

As it was, it was returning the number of expired nodes in total (which is very often much less than how many we actually remove). I believe this is what was thowing off the charting.. but I could be wrong there.